client/config/toml.go

```diff
@@ -70,18 +70,19 @@ func ensureConfFile(rootDir string) (string, error) {
 	return cfgPath, nil
 }
 
-func getClientConfig(cfgPath string) (*ClientConfig, error) {
-	viper.AddConfigPath(cfgPath)
-	viper.SetConfigName("client")
-	viper.SetConfigType("toml")
-	if err := viper.ReadInConfig(); err != nil {
+func getClientConfig(cfgPath string, v *viper.Viper) (*ClientConfig, error) {
```

Comment:
getClientConfig uses local instance of Viper to read config and unmarshal it to ClientConfig struct.